pkg/mcp/tools.go

```diff
@@ -63,6 +64,46 @@ var (
 		"wal_replay_status": {Tool: prometheusWalReplayTool, Handler: prometheusWalReplayToolHandler},
 	}
 
+	// thanosToolset contains all the tools to interact with thanos as a
+	// prometheus HTTP API compatible backend.
+	//
+	// Currently, the only difference between thanosToolset and
+	// prometheusToolset is that thanosToolset has the following tools
+	// removed because they are not implemented in Thanos and return 404s:
+	// [alertmanagers, config, wal_replay_status].
+	thanosToolset = map[string]server.ServerTool{
```

Comment:
It might be easier to understand the toolset diff between backends if we have a base set of tools and every backend imports the set and adds more to the list of tools available. That way we can prevent copy-pasta errors too

Reply:
Responded on slack already, but that's a good idea. Right now, I think it's small enough to be manageable and having it static is easy to validate. Allowing importing and overriding/extending toolsets would require it to be a runtime process and introduce new logic to handle it, with it's own potential bugs in logic or due to what might be over-complication. 

We'll see how much it annoys me/$theAuthor when the next provider gets implemented :wink: